### ISSUE TITLE
uniform call. argument for warnings / errors

### ISSUE THREAD

**Issue by lorenzwalthert** [jonmcalder/refactor#17]
Not sure whether it makes more sense to display where the error / warning occurred or not. I guess in interactive usage, it is pointless, but if you wrap for example `cfactor` in another function, it might be helpful?

**Comment by jonmcalder** [comment]
Agreed - where possible one should always make an effort to provide specific context for warnings and/or errors. Anything in particular you are referring to here or is it a more general comment?

**Comment by lorenzwalthert** [comment]
more general. But I don't know how to create the warnings dependent on whether for example `cfactor` was called interactively or from within a function...

**Comment by lorenzwalthert** [comment]
Also how to display the warning: E.g. when listing values: separate by comma or, new line?
